src/lib/kdmp-parser-structs.h

```diff
@@ -209,7 +212,7 @@ struct BMP_HEADER64 {
 
   uint64_t Pages;
 
-  uint8_t Bitmap[1];
+  std::array<uint8_t, 1> Bitmap; // TODO fix
```

Comment:
Hmm, might be better to not use `array` here as the size isn't known?

Reply:
AFAIK For modern C++ those 2 syntaxes (`u8 a[1]` and `std::array<u8, 1> a`) are strictly equivalent except that `std::array` allows the compiler to reason about the structure. But here we don't care about the compiler because the structure field is populated at runtime. 

![image](https://github.com/0vercl0k/kdmp-parser/assets/590234/90345815-4194-454a-be8a-0f2bbec0eb94)

Also removing `std::array` it will prevent `nanobind` exposing it and I'm not aware of a way around.

To be clear: there won't be any mem corruption at runtime as long as we still operate within the bounds of what's allocated for `Bitmap`.

Note: the TODO is misleading, I should've removed it a while back

Reply:
TODO removed with 1d20a49faeea996b2dae6f09a461ef726ab1a8d8